src/client.ts

```diff
@@ -7815,15 +7815,35 @@ export class MatrixClient extends TypedEventEmitter<EmittedEvents, ClientEventHa
      * Make a request to change your password.
      * @param {Object} authDict
      * @param {string} newPassword The new desired password.
+     * @param {boolean} logoutDevices Should all sessions be logged out after the password change. Defaults to true.
      * @param {module:client.callback} callback Optional.
      * @return {Promise} Resolves: TODO
      * @return {module:http-api.MatrixError} Rejects: with an error response.
      */
-    public setPassword(authDict: any, newPassword: string, callback?: Callback): Promise<any> { // TODO: Types
+    public setPassword(
+        authDict: any,
+        newPassword: string,
+        logoutDevices: boolean,
+        callback?: Callback
+        ): Promise<any>;
+    public setPassword(
+        authDict: any,
+        newPassword: string,
+        logoutDevices?: Callback | boolean,
+        callback?: Callback,
+    ): Promise<any> { // TODO: Types
+        if (typeof logoutDevices !== 'boolean') {
+            logoutDevices = true;
+        }
+        if (typeof logoutDevices === 'function') {
+            callback = logoutDevices;
+        }
```

Comment:
I think these might be backwards, clobbering the callback to be `true` in the backwards compat case

Reply:
Well spotted, thank you.